api/src/graphql/Post.ts

```diff
@@ -17,29 +58,75 @@ export const PostQuery = extendType({
       type: "Post",
```

Comment:
```suggestion
      t.nonNull.list.nonNull.field("posts", {
      type: "Post",
```
なんかsuggestionうまくいかなかったんですが、
全てのpostを取得するqueryに関してはfeed => postsでもいいかなと思ったりしました

Reply:
@sniper-fly これってfeedにされた理由あったりしますか？